geometry/geometry/triangles.plist

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>1</key>
+	<dict>
```

Comment:
I remember refactoring when we switched from dictionaries to arrays of points. It saves a lot of boilerplate code. Why are we returning back to dict?